pkg/nodeaddress/node_address.go

```diff
@@ -130,7 +142,12 @@ func InitDefaultPrefix(device string) {
 		}
 
 		ipv4AllocRange = ip4net
+		log.Infof("Automatically add IPv4 allocation %s", ipv4AllocRange)
+	}
 
+	if ipv6AllocRange == nil {
+		// The IPv6 allocation should be based from the IPv4 allocation
```

Comment:
The IPv6 allocation should be derived from the IPv4 allocation.

Reply:
Fixed